lib/maze/server.rb

```diff
@@ -153,13 +159,14 @@ def start
             end
 
             # When adding more endpoints, be sure to update the 'I should receive no requests' step
-            server.mount '/notify', Servlet, errors
-            server.mount '/sessions', Servlet, sessions
-            server.mount '/builds', Servlet, builds
-            server.mount '/uploads', Servlet, uploads
-            server.mount '/sourcemap', Servlet, sourcemaps
-            server.mount '/react-native-source-map', Servlet, sourcemaps
-            server.mount '/logs', LogServlet
+            server.mount '/notify', Servlets::Servlet, errors
+            server.mount '/sessions', Servlets::Servlet, sessions
+            server.mount '/builds', Servlets::Servlet, builds
+            server.mount '/uploads', Servlets::Servlet, uploads
+            server.mount '/sourcemap', Servlets::Servlet, sourcemaps
+            server.mount '/react-native-source-map', Servlets::Servlet, sourcemaps
+            server.mount '/commands', Servlets::CommandServlet
```

Comment:
From the client POV I think this should be `/command` (no plural) - i.e. `GET /command` to return the next command.

`GET /commands` would make sense if this returned an array of commands, which it does not.

Reply:
Agreed - changed.